packit_service/worker/reporting.py

```diff
@@ -87,6 +88,25 @@ def __set_pull_request_status(
                 uid=hashlib.md5(check_name.encode()).hexdigest(),
             )
 
+    def report_status_by_comment(
+        self,
+        state: CommitStatus,
+        url: str,
+        check_names: Union[str, list, None],
+    ):
+        """
+        Reporting build status with MR comment if no permission to the fork project
+        """
+
+        if isinstance(check_names, str):
+            check_names = [check_names]
+
+        comment_msg = [
+            f"[{check}]({url}): {state.name.upper()}" for check in check_names
+        ]
```

Comment:
We can use a table here (not need to be done here). The table looks better but only in case we don't need to provide any logs inside.

Reply:
Yup we don't, I have changed it!